src/packages/sdk/src/getGenerators.ts

```diff
@@ -147,11 +150,16 @@ export async function getGenerators({
   })
 
   if (dmmf.datamodel.models.length === 0) {
+// MongoDB needs extras for @id: @map("_id") @db.ObjectId
+    if (config.datasources.some((d) => d.provider.includes('mongodb'))) {
+      throw new Error(missingModelMessageMongoDB)
+    }
+
     throw new Error(missingModelMessage)
   }
 
   if (
-    config.datasources.some((d) => d.provider.includes('mongoDb')) &&
+    config.datasources.some((d) => d.provider.includes('mongodb')) &&
```

Comment:
Do we lowercase the providers?

Reply:
From what I tried uppercase even fails validation in the Prisma extension.